test/certs/webpki.go

```diff
@@ -38,7 +38,7 @@ func genKey(path string, inSlot string) error {
 	if err != nil {
 		return err
 	}
-	output, err := exec.Command("bin/ceremony", "-config", tmpPath).CombinedOutput()
+	output, err := exec.Command("go", "run", "./cmd/ceremony", "-config", tmpPath).CombinedOutput()
```

Comment:
Is this `go run` and relatedly the `runCeremony` function because the binary might not be built yet so we have to invoke the go runtime and compile ceremony just-in-time? If so, that's 6 distinct compilations which could account for the slowdown you mentioned offline.

Reply:
Yeah, I've changed this to compile once at the beginning, then rely on that binary. It's slightly faster, but not much. I think most of the time here is in the RSA keygens.